src/spinner.rs

```diff
@@ -0,0 +1,85 @@
+use yew::prelude::*;
+
+#[derive(PartialEq)]
+pub enum SpinnerSize {
+    Sm,
+    Md,
+    Lg,
+    Xl,
+}
+
+impl Default for SpinnerSize {
+    fn default() -> Self {
+        Self::Xl
```

Comment:
Looking and the docs, and I might be wrong, but it looks like the "default" is actually without a size class.

Reply:
Yes you're right, I've not noticed that. Thank!